### PR TITLE
Minio service guide

### PR DESCRIPTION
**Pull request by Mohini6649**
## Description
Updating minio service guide.

### CODE REVIEW

**Comment by realmbgl** [comment]
**quick start**
- from http://localhost/services/minio/0.1.0/quick-start-guide/#steps remove the repo add commands, jut say in words to add the edge-lb packages, edge-lb is a dcos enterprise feature so the repos should not be made public here
- the edgelb configuration should only contain the config for miniod
- same for the edge-lb tls config
- the tls aspect should be moved to the security section
- erasure coding stuff http://localhost/services/minio/0.1.0/quick-start-guide/#erasure-coding-schemes should be a section under configuration

**backup**
- talks about s3 compatible, but there is description on how you specify an s3 alternative endpoint

**troubleshootin/recover**
- we need to make it clear when this is necessary, for example after node replace

**Comment by ryadav88** [comment]
@SarahStegall @jpe442  can one of you review this PR to add service guide for Minio?

File: pages/services/include/getting-started.tmpl [hunk 1]
```diff
@@ -3,15 +3,15 @@ Getting started with a test instance of the DC/OS {{ model.techName }} service i
 ## Prerequisites
 
 - Depending on your security mode in Enterprise DC/OS, you may [need to provision a service account]({{ model.install.serviceAccountInstructionsUrl }}) before installing. Only someone with `superuser` permission can create the service account.
-	- `strict` [security mode](/latest/security/ent/#security-modes) requires a service account.
```
Comment (SarahStegall) [inline pages/services/include/getting-started.tmpl]: Do not ever, ever edit a template (tmpl) file. These files are used by many different service docs, not just this one. I am deleting all these changes and moving the material you added to the getting-started/index.md file. They will appear at the beginning of the file, before you call the getting-started.tmpl file. I am reverting all the changes in this file back to the original.

**Comment by ryadav88** [comment]
@SarahStegall this is now a duplicate of https://github.com/mesosphere/dcos-docs-site/pull/1785. Can this be closed?

**Pull request closed without merging**